src/liballoc/sync.rs

```diff
@@ -107,7 +107,8 @@ const MAX_REFCOUNT: usize = (isize::MAX) as usize;
 /// // The two syntaxes below are equivalent.
 /// let a = foo.clone();
 /// let b = Arc::clone(&foo);
-/// // a and b both point to the same memory location as foo.
+/// // a and b both point to the same memory location where foo resides
+/// // (not where the value wrapped by foo resides).
```

Comment:
In fact `a`, `b`, and `foo` are all separate `Arc`s pointing to the memory location where `foo` resides.

Reply:
As per your other comment, I agree, will remove this.